vital/types/rotation.cxx

```diff
@@ -318,8 +255,8 @@ compose_rotations( rotation_< T > const& platform_rotation,
   template VITAL_EXPORT rotation_< T > interpolate_rotation( rotation_< T > const & A, rotation_< T > const & B, T f ); \
   template VITAL_EXPORT void                                            \
   interpolated_rotations( rotation_< T > const & A, rotation_< T > const & B, size_t n, std::vector< rotation_< T > > &interp_rots ); \
-  template VITAL_EXPORT rotation_< T > compose_rotations( T p_y, T p_p, T p_r, T s_y, T s_p, T s_r ); \
```

Comment:
As mentioned elsewhere, It seems like it could be nice to keep this utility function to avoid writing boilerplate elsewhere.

Reply:
The one which takes rotations as input seems like it could be easily removed though.